ocfweb/docs/docs/staff/backend/prometheus.md

```diff
@@ -0,0 +1,44 @@
+[[!meta title="Prometheus"]]
+
+We use Prometheus to provide real-time monitoring of our hardware. The master is [[dementors|doc staff/backend/servers]] which
+uses the Node Exporter to collect data from other servers.
+
+We monitor servers, desktops, and staff VMs, but not the hozer boxes.
+Additionally, we don't receive email alerts for staff VMs. Monitoring for the networking switch, blackhole, is currently under development.
+
+## Alerts
+
+Alerts can be viewed at [prometheus.ocf.berkeley.edu/alerts](https://prometheus.ocf.berkeley.edu/alerts). They are configured at [this folder][prometheus-puppet] in the Puppet configs.
+
+Alerts are currently under development and may not be fully comprehensive.
+
+## Custom Queries
+
+Prometheus supports querying a wide variety of metrics. (For a full list, go to [Prometheus](https://prometheus.ocf.berkeley.edu) and use the "insert metric at cursor" dropdown.) A basic query comes in the form:
+```
+metric{label="value", label2="value2", ...}
+```
+For example, if you would like to view the total RAM installed on each of the [[servers|doc staff/backend/servers]] you can query `node_memory_Active_bytes{host_type="server"}`.
+
+To view the per-second rate of a metric, use
+```
+rate(metric{label="value",...})
+```
+For example, the data sent in bytes/second over the past 5 minutes by `fallingrocks` can be retrieved using `rate(node_network_transmit_bytes_total{instance="fallingrocks"}`.
+
+Queries are best used in conjunction with Grafana, as to produce more readable results and save them for future reference. The next section will give more details on how to do this.
+
+## Grafana
+
+The frontend for Prometheus is [Grafana][grafana], which displays statistics collected by Prometheus in a user-friendly manner. Some of the more useful dashboards available are:
```

Comment:
We have ocf.io/printers as a shortcut for the printers dashboard. It could be useful to make shortlinks for the other dashboards, so people don't need to go to the docs to get links to them.